react/package.json

```diff
@@ -33,5 +33,10 @@
     "vtex.render-runtime": "http://vtex.vtexassets.com/_v/public/typings/v1/vtex.render-runtime@8.126.5/public/@types/vtex.render-runtime",
     "vtex.styleguide": "http://vtex.vtexassets.com/_v/public/typings/v1/vtex.styleguide@9.134.1/public/@types/vtex.styleguide"
   },
+  "jest": {
+    "setupFilesAfterEnv": [
+      "<rootDir>/setupTests.js"
+    ]
+  },
```

Comment:
> I thought that maybe changing this would fix the issues you posted 😄 but if it doesn't, I don't mind keeping it as is

@lucasecdb, this would be equivalent to renaming, no? But now I get your point

Reply:
ohh, it already is a after env setup file, right? yeah, the rename wouldn't matter then 😂

Reply:
I tested it anyway and it does not solve :(
but ty for the suggestion

Reply:
(btw sent last message before seeing your reply :slowpoke:)